tests/base.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2015-2017 Bitergia
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA.
+#
+# Authors:
+#     Valerio Cosentino <valcos@bitergia.com>
+#     Santiago Dueñas <sduenas@bitergia.com>
+#
+
+import os
+import shutil
+import tempfile
+import unittest
+
+from perceval.archive import Archive
+
+
+class TestCaseBackendArchive(unittest.TestCase):
+    """Unit tests for Backend using the archive"""
+
+    def setUp(self):
+        self.test_path = tempfile.mkdtemp(prefix='perceval-mozilla_')
```

Comment:
I changed this `prefix` from `perceval_` to `perceval-mozilla_`. This is the only change from the original `base.py` script found in the main repository of Perceval.